exercises/react/tests/react.rs

```diff
@@ -99,20 +99,41 @@ fn error_setting_a_compute_cell() {
     assert!(reactor.set_value(output, 3).is_err());
 }
 
+struct Expector {
+    value: std::cell::Cell<Option<isize>>,
```

Comment:
I'm only doing this because I thought If I just use `cell` here (with a `use std::cell::Cell` before, of course), what if a submitted implementation (which we `use react::*`) has a `Cell` ? will the conflicting names named in a `use` cause problems? I admit I never tried.

Reply:
It does **not** cause problems if `Cell` is specifically named:

```rust
mod m1 {
    #[derive(Debug)]
    pub struct A {
        pub a: usize,
    }
}

mod m2 {
    #[derive(Debug)]
    pub struct A {
        pub a: usize,
    }
}

use m1::*;
use m2::A;

fn main() {
    println!("Hello, world! {:?}", A{a: 1});
}
```

So, could go back to `use std::cell::Cell`, but perhaps I'll leave it this way for clarity in case someone doesn't know the rules. I didn't know off the top of my head.